mysql-test/suite/innodb/t/innodb_buffer_pool_resize_temporary.test

```diff
@@ -24,4 +24,4 @@ SELECT COUNT(*),MIN(a),MAX(a) FROM t1;
 DROP TEMPORARY TABLE t1;
 
 SET GLOBAL innodb_limit_optimistic_insert_debug=@save_limit;
-SET GLOBAL innodb_buffer_pool_size=@save_size;
+SET GLOBAL innodb_buffer_pool_size=@save_size;
```

Comment:
Each file should remain terminated by a newline, unless there is a strong reason to change that.